src/masoniteorm/query/grammars/BaseGrammar.py

```diff
@@ -253,11 +253,15 @@ def process_order_by(self):
             self
         """
         sql = ""
-        for order_bys in self._order_by:
-            column, direction = order_bys
-            sql += self.order_by_string().format(
-                column=self._table_column_string(column), direction=direction.upper()
-            )
+        if self._order_by:
+            for order_bys in self._order_by:
+                if len(sql):
+                    sql += ", "
+                column, direction = order_bys
+                sql += self.order_by_string().format(
+                    column=self._table_column_string(column), direction=direction.upper()
+                )
+            sql = "ORDER BY " + sql
```

Comment:
Grammar classes should abstract all SQL. Although all of the databases we currently support are the same, Databases we may support in the future may not. it really should be calling the child grammar classes to get this (`self.order_by_string()`)